.travis.yml

```diff
@@ -10,16 +14,28 @@ addons:
   postgresql: "9.5"
 
 install:
-  - pip install -r requirements.txt
-  - pip install pycodestyle
+  - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then 
+      pip install -r requirements.txt;
+      pip install pycodestyle;
+    else
```

Comment:
I am not sure I understand why 3.6 is needed and what it does?

Reply:
The thing is scancode works on python2.7 while coala works on python3+ versions. So to cater to both of them I added this workaround, which will use run travis on 2 different versions of python, 2.7  for scancode and 3.6 for coala.